pom.xml

```diff
@@ -332,35 +332,37 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <version>${mockito.version}</version>
+        <scope>test</scope>
+      </dependency>
     </dependencies>
   </dependencyManagement>
 
   <build>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>io.cdap</groupId>
-          <artifactId>cdap-maven-plugin</artifactId>
-          <version>1.1.0</version>
-          <configuration>
-            <cdapArtifacts>
-              <parent>system:cdap-data-pipeline[6.1.1,7.0.0-SNAPSHOT)</parent>
-              <parent>system:cdap-data-streams[6.1.1,7.0.0-SNAPSHOT)</parent>
-            </cdapArtifacts>
-          </configuration>
-          <executions>
-            <execution>
-              <id>create-artifact-config</id>
-              <phase>prepare-package</phase>
-              <goals>
-                <goal>create-plugin-json</goal>
-              </goals>
-            </execution>
-          </executions>
-        </plugin>
-      </plugins>
-    </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>io.cdap</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
```

Comment:
this and maven-bundle-plugin should be in the pluginManagement section, because not all of the submodules are plugins (database-commons). The plugin submodules should have a build section while the common ones do not.

Reply:
Good catch! Didn't notice that `database-commons` doesn't have this section.

Reply:
done:)